mathics/core/symbols.py

```diff
@@ -194,20 +195,20 @@ def __cmp(self, other) -> int:
         return 0
 
 
-class Atom(BaseElement):
+class Atom(BaseElement, ImmutableValueMixin):
```

Comment:
``Symbol`` is a subclass of ``Atom``, but must be evaluated. Maybe the feature ``InmutableValueMinin`` must be added to the subclasses (``Number``, ``String``, ``BitArray``, etc)

Reply:
Yes, it _should_ be added to those, and I haven't done those yet, but I will add them.

Reply:
My point is that (at the end) ``Atom`` shouldn't be ``ImmutableValueMixin``.

Reply:
You are correct. It has been removed now. Thanks!